Mage/src/main/java/mage/cards/mock/MockCard.java

```diff
@@ -101,6 +103,7 @@ public MockCard(final MockCard card) {
         this.adventureSpellName = card.adventureSpellName;
         this.isModalDoubleFacesCard = card.isModalDoubleFacesCard;
         this.manaValue = card.manaValue;
+        this.extraDeckCard = card.extraDeckCard;
```

Comment:
Redundant since this is done in `super()`, right?

Reply:
yup, good catch